detect_variants/tumor_only_detect_variants.cwl

```diff
@@ -104,7 +126,7 @@ steps:
     bam_readcount:
         run: ../pvacseq/bam_readcount.cwl
```

Comment:
bam_readcount.cwl output changed in https://github.com/genome/cancer-genomics-workflow/pull/223